changelogs/fragments/1475-xfconf-facts.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - xfconf - added option ``disable_facts`` to disable facts and its associated deprecation warning (https://github.com/ansible-collections/community.general/issues/1475).
+deprecated_features:
+  - xfconf - returning output as facts is deprecated, should be removed by version 4.0.0
```

Comment:
```suggestion
  - xfconf - returning output as facts is deprecated, this will be removed in community.general 4.0.0. Please register the task output in a variable and use it instead. You can already switch to the new behvaior now by using the new ``disable_facts`` option (https://github.com/ansible-collections/community.general/pull/1747).
```

Reply:
```suggestion
  - xfconf - returning output as facts is deprecated, this will be removed in community.general 4.0.0. Please register the task output in a variable and use it instead. You can already switch to the new behavior now by using the new ``disable_facts`` option (https://github.com/ansible-collections/community.general/pull/1747).
```

Reply:
@felixfontein there is a small typo in your suggestion, hence the other one.